features/raadpleeg-met-reisdocumentnummer/fout-cases.feature

```diff
@@ -78,73 +75,22 @@ Rule: De reisdocumentnummer parameter bevat een lijst van maximaal 1 reisdocumen
     | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
     | title    | Een of meerdere parameters zijn niet correct.               |
     | status   | 400                                                         |
-    | detail   | De foutieve parameter(s) zijn: gemeenteVanInschrijving.     |
-    | code     | paramsValidation                                            |
-    | instance | /haalcentraal/api/reisdocumenten/reisdocumenten             |
-    En heeft het object de volgende 'invalidParams' gegevens
-    | code     | name                    | reason                  |
-    | required | gemeenteVanInschrijving | Parameter is verplicht. |
-
-Rule: Een gemeenteVanInschrijving is een string bestaande uit exact 4 cijfers
-
-  @fout-case
-  Abstract Scenario: <titel>
-    Als reisdocumenten wordt gezocht met de volgende parameters
-    | naam                    | waarde                         |
-    | type                    | RaadpleegMetReisdocumentnummer |
-    | reisdocumentnummer      | NE3663258                      |
-    | gemeenteVanInschrijving | <gemeenteVanInschrijving>      |
-    | fields                  | reisdocumentnummer             |
-    Dan heeft de response een object met de volgende gegevens
-    | naam     | waarde                                                      |
-    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
-    | title    | Een of meerdere parameters zijn niet correct.               |
-    | status   | 400                                                         |
-    | detail   | De foutieve parameter(s) zijn: gemeenteVanInschrijving.     |
+    | detail   | De foutieve parameter(s) zijn: reisdocumentnummer.          |
     | code     | paramsValidation                                            |
     | instance | /haalcentraal/api/reisdocumenten/reisdocumenten             |
     En heeft het object de volgende 'invalidParams' gegevens
-    | code    | name                    | reason                                      |
-    | pattern | gemeenteVanInschrijving | Waarde voldoet niet aan patroon ^[0-9]{4}$. |
-
-    Voorbeelden:
-    | gemeenteVanInschrijving | titel                                                                          |
-    | 123                     | De opgegeven gemeenteVanInschrijving is een string met minder dan vier cijfers |
-    | 12345                   | De opgegeven gemeenteVanInschrijving is een string met meer dan vier cijfers   |
-    | 123O                    | De opgegeven gemeenteVanInschrijving is een string met een letter              |
-    | 12.4                    | De opgegeven gemeenteVanInschrijving bevat niet-cijfer                         |
+    | code     | name               | reason                        |
+    | maxItems | reisdocumentnummer | Array bevat meer dan 1 items. |
 
 Rule: Alleen gespecificeerde parameters bij het opgegeven zoektype mogen worden gebruikt
 
```

Comment:
hieronder in regel 103 wordt parameter gemeenteVanInschrijving gebruikt

Reply:
verwijderd